e2e-tests/cypress/support/step_definitions/send_coin_steps.ts

```diff
@@ -0,0 +1,69 @@
+import { And, Then, When } from '@badeball/cypress-cucumber-preprocessor'
+import { SendPage } from '../../e2e/models/SendPage'
+
+const sendPage = new SendPage()
+
+When(
+  'the user fills the send form with {string} {string} {string}',
+  (email: string, amount: string, memoText: string) => {
+    sendPage.enterReceiverEmail(email)
+    sendPage.enterAmount(`${amount}`)
+    sendPage.enterMemoText(memoText)
+  }
+)
+
+And('the user submits the send form', () => {
+  sendPage.submit()
+  cy.get(sendPage.confirmationBox).should('be.visible')
+})
+
+
+Then('the transaction details are presented for confirmation', () => {
+  cy.get('.transaction-confirm-send').contains('raeuber@hotzenplotz.de')
+  cy.get('.transaction-confirm-send').contains('+ 120.50 GDD')
+  cy.get('.transaction-confirm-send').contains('Some memo text')
+  cy.get('.transaction-confirm-send').contains('+ 515.11 GDD')
+  cy.get('.transaction-confirm-send').contains('− 120.50 GDD')
+  cy.get('.transaction-confirm-send').contains('+ 394.61 GDD')
+})
```

Comment:
Should we not define each expected value over parameters?
```suggestion
Then('the transaction details are presented for confirmation', (receiverEmail: string, sendAmount: string, sendMemoText: string, senderBalance: string, newSenderBalance: string) => {
  cy.get('.transaction-confirm-send').contains(receiverEmail)
  cy.get('.transaction-confirm-send').contains('+ ' + sendAmount + ' GDD')
  cy.get('.transaction-confirm-send').contains(sendMemoText)
  cy.get('.transaction-confirm-send').contains('+ ' + senderBalance + ' GDD')
  cy.get('.transaction-confirm-send').contains('− ' + sendAmount + ' GDD')
  cy.get('.transaction-confirm-send').contains('+ ' + newSenderBalance + ' GDD')
})
```